script/tool/test/list_command_test.dart

```diff
@@ -29,17 +29,17 @@ void main() {
       runner.addCommand(command);
     });
 
-    test('lists plugins', () async {
-      createFakePlugin('plugin1', packagesDir);
+    test('lists top-level packages', () async {
+      createFakePackage('package1', packagesDir);
       createFakePlugin('plugin2', packagesDir);
```

Comment:
was leaving plugin2 (and later plugin3) intentional?

Reply:
Yes, it was to create a mix. Ideally more tests would be using a mix of plugins and packages just to make sure there aren't any unintended behaviors for one type but not the other.